src/RamValue.h

```diff
@@ -17,9 +17,8 @@
 #pragma once
 
 #include "BinaryFunctorOps.h"
-#include "RamIndex.h"
 #include "RamNode.h"
-#include "RamRecords.h"
+// #include "RamRecords.h"
```

Comment:
Remove this.

Reply:
I will fix this in the next pull-request.